static/styles/message.css

```diff
@@ -0,0 +1,128 @@
+$first_column_width: 46px;
+$distance_of_text_elements_from_message_box_top: 8px;
+
+.message_row {
+    &.include-sender {
+        .messagebox .messagebox-content {
+            grid-template-rows: 25px repeat(3, auto);
+            padding-top: 4px;
+
+            .message_content {
+                padding: 0;
+                grid-row-start: 2;
+            }
+
+            .message_time {
+                align-self: center;
+                position: unset;
+                margin-top: 1px;
+            }
+
+            .message_edit_notice {
+                align-self: center;
+                top: 2px;
+            }
+
+            .message_controls {
+                align-self: center;
+                position: unset;
+            }
+
+            .message_sender {
+                overflow: hidden;
+                text-overflow: ellipsis;
+                grid-column: 1 / 3;
+                grid-row: 1 / 2;
+
+                > span {
+                    display: flex;
+                }
+
+                .inline_profile_picture {
+                    flex-shrink: 0;
+                    /* Let user profile take extra height without
```

Comment:
Do you mean "user profile picture" here? The "user profile" is a modal.

Reply:
Yes. Fixed.